src/T2DMap.cpp

```diff
@@ -2645,61 +2644,61 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
             moveRoom->setToolTip(tr("Move room"));
             connect(moveRoom, SIGNAL(triggered()), this, SLOT(slot_moveRoom()));
 
-            auto deleteRoom = new QAction("delete", this);
+            auto deleteRoom = new QAction(tr("Delete", "Menu option to delete a room in the mapper"), this);
             deleteRoom->setToolTip(tr("Delete room"));
             connect(deleteRoom, SIGNAL(triggered()), this, SLOT(slot_deleteRoom()));
 
-            auto recolorRoom = new QAction("color", this);
+            auto recolorRoom = new QAction(tr("Color", "Menu option to change a room color in the mapper"), this);
             recolorRoom->setToolTip(tr("Change room color"));
             connect(recolorRoom, SIGNAL(triggered()), this, SLOT(slot_changeColor()));
 
-            auto spreadRooms = new QAction("spread", this);
+            auto spreadRooms = new QAction(tr("Spread", "Menu option to space out rooms in the mapper"), this);
             spreadRooms->setToolTip(tr("Increase map X-Y spacing for the selected group of rooms"));
             connect(spreadRooms, SIGNAL(triggered()), this, SLOT(slot_spread()));
 
-            auto shrinkRooms = new QAction("shrink", this);
+            auto shrinkRooms = new QAction(tr("Shrink", "Menu option to move rooms closer in the mapper"), this);
             shrinkRooms->setToolTip(tr("Decrease map X-Y spacing for the selected group of rooms"));
             connect(shrinkRooms, SIGNAL(triggered()), this, SLOT(slot_shrink()));
 
-            auto lockRoom = new QAction("lock", this);
+            auto lockRoom = new QAction(tr("Lock", "Menu option to lock a room for speed walks in the mapper"), this);
             lockRoom->setToolTip(tr("Lock room for speed walks"));
             connect(lockRoom, SIGNAL(triggered()), this, SLOT(slot_lockRoom()));
 
-            auto unlockRoom = new QAction("unlock", this);
+            auto unlockRoom = new QAction(tr("Unlock", "Menu option to unlock a room for speed walks in the mapper"), this);
             unlockRoom->setToolTip(tr("Unlock room for speed walks"));
             connect(unlockRoom, SIGNAL(triggered()), this, SLOT(slot_unlockRoom()));
 
-            auto weightRoom = new QAction("weight", this);
+            auto weightRoom = new QAction(tr("Weight", "Menu option to set a room wight in the mapper"), this);
             weightRoom->setToolTip(tr("Set room weight"));
             connect(weightRoom, SIGNAL(triggered()), this, SLOT(slot_setRoomWeight()));
 
-            auto roomExits = new QAction("exits", this);
+            auto roomExits = new QAction(tr("Exits", "Menu option to set exits for a room in the mapper"), this);
             roomExits->setToolTip(tr("Set room exits"));
             connect(roomExits, SIGNAL(triggered()), this, SLOT(slot_setExits()));
 
-            auto roomSymbol = new QAction("symbol", this);
+            auto roomSymbol = new QAction(tr("Symbol", "Menu option to set symbol(s) to mark rooms in the mapper"), this);
```

Comment:
Technically I call the combination of graphemes when put together a single "symbol" - so perhaps this should read:
`"Menu option to set one or more characters as a symbol to mark rooms in the mapper"`

Reply:
I wouldn't want to dive into graphemes, characters, symbols, etc. at this point.

Reply:
It does have a bearing on how the text/`QString` entered is handled - together the combination now becomes the room ***symbol*** as such and is handled like that for the (room symbol) display in the extra display accessible from the preferences - e.g. if the text entered is "Shop" then the *symbol* for the rooms concerned is *all* of those characters in that order `Shop`...

Reply:
Lets see how it works out then...